acceptancetests/jujupy/client.py

```diff
@@ -2121,6 +2122,19 @@ def switch(self, model=None, controller=None):
             raise ValueError('No target to switch to has been given.')
         self.juju('switch', (':'.join(args),), include_e=False)
 
+    def scp(self, _from, _to):
+        args = ('-m', self.model_name)
+        args += (_from, _to)
+        self.juju('scp', args)
+
+
+class K8sClient(ModelClient):
+    ...
+
+
+class CaasClient(ModelClient):
+    ...
+
```

Comment:
What is expected to be contained in these 2 classes? Does it make sense to subclass ModelClient?

Is a K8sClient an extension of a juju model? Same question for CaasClient. How does a user interact with a CaasClient of K8sClient. 

I suspect a CaasClient is very much like a ModelClient, what makes then different? Are they interchangable?